dhcp/dhcp.py

```diff
@@ -583,6 +583,19 @@ def sorted_hosts(hosts):
 IP    = b'\x45\x00\x00\x00\x00\x00\x40\x00\x40\x11\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
 ETHER = b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x08\x00'
 
+def getmac():
+    try:
+        yamlFile = open('/home/cs158b/batman/piman.yaml')
```

Comment:
The path to the yaml should probably be team-agnostic. Any way to get the path name from a configurable source rather than have it hard coded?

Reply:
Oh yea that just slipped my mind, good catch Ill will look into that.